addons/website/views/website_templates.xml

```diff
@@ -341,6 +323,71 @@
 </template>
 
 <!-- Header templates -->
+<template id="template_header_default" inherit_id="website.layout" name="Template Header Default" active="True">
+        <xpath expr="//header//nav" position="replace">
+        <nav class="navbar navbar-expand-lg navbar-light o_colored_level o_cc shadow-sm">
```

Comment:
- Indentation issue
- Probably remove the shadow-sm class, it conflicts with the shadow option no?

Reply:
1000 lines of code. 1 line badly indented. You found it 🥇 
As discussed on Discord, the shadow option is not a priority but we should make it work.